### PR TITLE
Add Spanish translation for URLs Lead The Way.

### PR DESCRIPTION
**Pull request by mblayman**
This is an example PR to show how to translate an Understand Django
article. I've tried to break down the process into each commit. I will
write up a translation guide, but I hope that this example will help as
an aide.

### CODE REVIEW

File: content/understand-django/2020-01-22-urls-lead-way.es.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
+---
+title: "Las URL Marcan El Camino"
+slug: "url-marcan-camino"
+description: >-
+    How does a Django site know
```
Comment (mblayman) [inline content/understand-django/2020-01-22-urls-lead-way.es.md:5]: The description is missing a translation.

File: content/understand-django/2020-01-22-urls-lead-way.es.md [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+]
+```
+
+El uso de `include` le da a cada aplicación de Django autonomía en qué vistas necesita definir. El proyecto puede ser felizmente "ignorante" de lo que está haciendo la aplicación.
+
+Adicionalmente, se elimina la repetición de `schools/` o `students/` del primer ejemplo. A medida que Django procesa una ruta, coincidirá con la primera parte de la ruta y pasará el *resto* a la URLconf que se define en la aplicación individual. De esta forma, las configuraciones de URL pueden formar un árbol donde la raíz URLconf es donde comienzan todas las solicitudes, pero las aplicaciones individuales pueden manejar los detalles a medida que una solicitud se enruta a la aplicación adecuada.
+
+## Nombrar URLs
+
+Hemos analizado las principales formas en que las URL se definen con `path`, `re_path`, e `include`. Hay otro aspecto a considerar. ¿Cómo podemos referirnos a las URL en otros lugares del código? Considere esta vista (bastante tonta):
```
Comment (mblayman) [inline content/understand-django/2020-01-22-urls-lead-way.es.md:293]: Was this a mistake? I don't know what `e` means by itself in Spanish.
```suggestion
Hemos analizado las principales formas en que las URL se definen con `path`, `re_path`, y `include`. Hay otro aspecto a considerar. ¿Cómo podemos referirnos a las URL en otros lugares del código? Considere esta vista (bastante tonta):
```

File: content/understand-django/2020-01-22-urls-lead-way.es.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+    path("", views.index, name="index"),
+    path(
+        "<int:school_id>/",
+        views.school_detail,
+        name="detail"
+    ),
+]
+```
+
+By adding `app_name`,
```
Comment (mblayman) [inline content/understand-django/2020-01-22-urls-lead-way.es.md:412]: This section was missing the translation in the Google Doc.

File: content/understand-django/2020-01-22-urls-lead-way.es.md (hunk 2 above)
Comment (mblayman) [reply content/understand-django/2020-01-22-urls-lead-way.es.md:293]: Neat! Thanks for educating me on this. I took a few years of Spanish in high school and never encountered this rule. It's fascinating. I guess it's similar to how English uses `an` in front of a word that starts with a vowel sound.

Comment (HappyCoder77) [reply content/understand-django/2020-01-22-urls-lead-way.es.md:293]: Exactly!

File: content/understand-django/2020-01-22-urls-lead-way.es.md (hunk 1 above)
Comment (HappyCoder77) [reply content/understand-django/2020-01-22-urls-lead-way.es.md:5]: ¿Cómo sabe un sitio de Django dónde enviar solicitudes? ¡Tienes que decírselo!
    En este artículo que continúa la serie Comprendiendo Django, examinaremos las URL y cómo permitir que tus usuarios lleguen al lugar correcto.

File: content/understand-django/2020-01-22-urls-lead-way.es.md (hunk 3 above)
Comment (HappyCoder77) [reply content/understand-django/2020-01-22-urls-lead-way.es.md:412]: Al añadir app_name,

Comment (mblayman) [reply content/understand-django/2020-01-22-urls-lead-way.es.md:412]: Sorry, I know you haven't done much with PRs so I should have been more clear. The "section" is lines 412-419. It appears that the translation you listed is just for 412.

Comment (HappyCoder77) [reply content/understand-django/2020-01-22-urls-lead-way.es.md:406]: Al añadir app_name, señalamos a Django que esas vistas están en un espacio de nombres. Ahora cuando queramos obtener una URL, usaremos el nombre del espacio de nombres y la url unidos con dos puntos.

**Pull request merged**